java/src/client/facade/FacadeManager.java

```diff
@@ -0,0 +1,65 @@
+package client.facade;
+
+import shared.models.game.ClientModel;
+
+/**
+ * Created by audakel on 9/17/16.
```

Comment:
Unneeded. Authorship of code is for team members in development only, not for public usage.